src/Contexts/Adaptors/SubstrateHomeAdaptor.tsx

```diff
@@ -297,6 +298,37 @@ export const SubstrateHomeAdaptorProvider = ({
     [api, setDepositNonce, setTransactionStatus, address, homeChainConfig]
   );
 
+  const handleCheckSupplies = useCallback(
+    async (
+      amount: number,
+      tokenAddress: string,
+      destinationChainId: number
+    ) => {
+      if (homeChainConfig) {
+        const destinationChainConfig = getСhainConfig(destinationChainId);
+        const token = homeChainConfig.tokens.find(
+          (token) => token.address === tokenAddress
+        );
+
+        if (destinationChainConfig?.type === "Ethereum" && token) {
+          return await hasTokenSupplies(
+            destinationChainConfig,
+            tokens,
+            token,
+            amount,
+            tokenAddress
+          );
+        } else {
+          console.warn(
+            `The destination chain type ${destinationChainConfig?.type} is unknown. Please check it.`
```

Comment:
Add in the beginning `Liquidity check is skipping.`

Reply:
fixed